gemfirexd/tools/src/main/java/com/pivotal/gemfirexd/tools/internal/MiscTools.java

```diff
@@ -120,7 +120,7 @@ public final void addCommandOptions(final Options opts) {
           addPathOption(opts);
           addEncodingOption(opts);
           addIgnoreErrorsOption(opts);
-//          addParamOption(opts);
+          addParamOption(opts);
```

Comment:
Why has addNumTimesToRun been removed?

Also refactor the addConnectionOptions in ToolsBase to split out the ones used here (without locators/mcast options) into a separate method so that override here can call that method instead of duplicating the code.

Reply:
As SNAP-3006 asked to sync up "run" command's "usage" with Snappydata docs, removed addNumTimesToRun. 
Now reverted back the changes, and only "-locators" has been hide from "usage". For that as suggested split out addConnectionOptions into a separate method addOptionalConnectionOptions.